dashboard/lib/services/i18n/curriculum_sync_utils/serializers.rb

```diff
@@ -60,7 +60,7 @@ class LessonActivityCrowdinSerializer < CrowdinSerializer
       end
 
       class ResourceCrowdinSerializer < CrowdinSerializer
-        attribute :name
+        attributes :name, :url
```

Comment:
this might be a broader question but resources also have download_urls. Do we also plan to translate those the same way?

Reply:
@dju90 thoughts?

Reply:
ah that's new and different from Curriculum Builder, yes? How are they different from the links to the Google doc resource itself? In any case, I think, yes, we should plan to "translate" those links in the same way - wouldn't want to link folks to download an untranslated version of a resource

Reply:
These two examples are for videos...are there any download links for worksheets and the like (which I believe are all google docs or google slides themselves)?

Reply:
Oh good point. Looks like there aren't currently any resources that have download links to google docs/slides, so this point is pretty moot for now. I think for docs/slides it's unlikely to have a download link as the resources are downloadable from Drive but we don't prevent download links from being added for those types of resources.